ep/build-velox/src/get_velox.sh

```diff
@@ -2,8 +2,8 @@
 
 set -exu
 
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=main
+VELOX_REPO=https://github.com/ulysses-you/velox.git
```

Comment:
Please help to revert this change. Thanks.

Reply:
reverted